examples/src/bin/tlsserver-mio.rs

```diff
@@ -551,15 +555,29 @@ fn load_ocsp(filename: &Option<String>) -> Vec<u8> {
     ret
 }
 
+fn load_crls(filenames: &Vec<String>) -> Vec<UnparsedCertRevocationList> {
+    let mut ret = Vec::new();
```

Comment:
Naming nit: maybe name this `crls`?

Reply:
Done.

Reply:
I meant the `ret`. Did you rename something else? Actually, I think you could now write this as an iterator chain? `filenames.into_iter().map(|name| UnparsedCertRevocationList(fs::read(filename).expect("")).collect()`?

Reply:
> I meant the ret. Did you rename something else?

Yeah I misunderstood and renamed `load_crls` to `crls`. Sorry about that. I restored the original name to match the other `load_xx` fns nearby.

> I think you could now write this as an iterator chain?

Done, but it ended up a little bit messier than suggested because of needing a mutable buf for reading the DER into. Also clippy wanted `iter` instead of `into_iter` and the `filenames` arg to be `&[String]` instead of `Vec<String>`:

```rust
fn load_crls(filenames: &[String]) -> Vec<UnparsedCertRevocationList> {
    filenames
        .iter()
        .map(|filename| {
            let mut der = Vec::new();
            fs::File::open(filename)
                .expect("cannot open CRL file")
                .read_to_end(&mut der)
                .unwrap();
            UnparsedCertRevocationList(der)
        })
        .collect()
}
```